models/ngspice/testing/regression/resistor_r/device_netlists/2term_res_a.spice

```diff
@@ -1,44 +0,0 @@
-**************************
```

Comment:
Why are the resistor regression tests being removed?

Reply:
They are not removed. They are moved and changed.

Reply:
what's the new equivalent tests?